test/unit/_macros.js

```diff
@@ -81,6 +79,37 @@ export const quote = test.macro({
     t.is(actual, expected);
   },
   title(_, { input, shellName }) {
+    input = input
+      .replace(/\0/gu, "\\u{0000}")
+      .replace(/\t/gu, "\\t")
+      .replace(/\n/gu, "\\n")
+      .replace(/\v/gu, "\\v")
+      .replace(/\f/gu, "\\f")
+      .replace(/\r/gu, "\\r")
+      .replace(/\u0008/gu, "\\u{0008}")
+      .replace(/\u001B/gu, "\\u{001B}")
+      .replace(/\u0085/gu, "\\u{0085}")
+      .replace(/\u009B/gu, "\\u{009B}")
+      .replace(/\u00A0/gu, "\\u{00A0}")
+      .replace(/\u1680/gu, "\\u{1680}")
+      .replace(/\u2000/gu, "\\u{2000}")
+      .replace(/\u2001/gu, "\\u{2001}")
+      .replace(/\u2002/gu, "\\u{2002}")
+      .replace(/\u2003/gu, "\\u{2003}")
+      .replace(/\u2004/gu, "\\u{2004}")
+      .replace(/\u2005/gu, "\\u{2005}")
+      .replace(/\u2006/gu, "\\u{2006}")
+      .replace(/\u2007/gu, "\\u{2007}")
+      .replace(/\u2008/gu, "\\u{2008}")
+      .replace(/\u2009/gu, "\\u{2009}")
+      .replace(/\u200A/gu, "\\u{200A}")
+      .replace(/\u2028/gu, "\\u{2028}")
+      .replace(/\u2029/gu, "\\u{2029}")
+      .replace(/\u202F/gu, "\\u{202F}")
+      .replace(/\u205F/gu, "\\u{205F}")
+      .replace(/\u3000/gu, "\\u{3000}")
+      .replace(/\uFEFF/gu, "\\u{FEFF}");
+
     return `quote '${input}' for ${shellName}`;
   },
```

Comment:
Can duplication between this and :arrow_down: be avoided?:

https://github.com/ericcornelissen/shescape/blob/6d4b4cbb9152754d9e3fa84d722af6743f4bcd93/test/unit/_macros.js#L26-L60

Reply:
Resolved with 07567f35d5fbc1c56e45f0c9153b4dfa7dd81d8d